src/pages/About.js

```diff
@@ -0,0 +1,2 @@
+const About = () => <div>Hello from about page</div>;
+export default About;
```

Comment:
- [ ] kindly for better design change the location of your text to center maybe

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.